cmd/libvirt-provider/app/app.go

```diff
@@ -222,20 +223,28 @@ func Run(ctx context.Context, opts Options) error {
 
 	// Check if apinetKubeconfig is provided
 	var apinetClient client.Client
```

Comment:
Do we still need this variable here?

Reply:
As the client creation is not part of the if condition anymore, we don't need that variable anymore, too.
Thanks for that hint! I have removed it.